File: Client/Assets/Scripts/Integration/AuthorizationResults.cs

using System;

public enum AuthorizationResults : byte
{
	AUTH_CONNECTION_SUCCESS    = 1,
	AUTH_ALREADY_CONNECTED     = 2,
	AUTH_BANNED                = 3,
	AUTH_NOT_REGISTERED        = 4,
	AUTH_DISCONNECTED          = 5,
	AUTH_REGISTRATION_SUCCESS  = 6,
	AUTH_ALREADY_EXIST         = 7,
}

File: Server/src/main/java/org/jgame/udp/AuthorizationResults.java
package org.jgame.udp;

public interface AuthorizationResults {
    byte AUTH_CONNECTION_SUCCESS    = 1;
    byte AUTH_ALREADY_CONNECTED     = 2;
    byte AUTH_BANNED                = 3;
    byte AUTH_NOT_REGISTERED        = 4;
    byte AUTH_DISCONNECTED          = 5;
    byte AUTH_REGISTRATION_SUCCESS  = 6;
    byte AUTH_ALREADY_EXIST         = 7;
}
